cli-tips.mkd

```diff
@@ -1,8 +1,8 @@
-# placeholder for cli-tips
+# Placeholder for cli-tips
```

Comment:
I will accept this PR but I have to warn you that lowercase thing is a particular idiosyncracy of mine.  God knows when I picked it up, but it's ingrained now.  I will make an effort to fix it in other places also, over time, I promise :)

Reply:
Since, I'm going through this and git-notes repo content so, if you don't mind; can I make some more pull request for this?